01_homework/hw3_task8.cpp

```diff
@@ -0,0 +1,84 @@
+#include <vector>
+#include <thread>
+#include <iterator>
+#include <numeric>
+#include <iostream>
+#include <cassert>
+#include <future>
+
+template<typename Iterator>
+struct accumulate_block
+{
+    void operator()(Iterator first, Iterator last)
+    {
+        return std::accumulate(first, last, 0);
```

Comment:
Tak w pełni poprawnie to tutaj powinno być T{} zamiast 0

Reply:
wydawalo mi sie ze dodalem, musialem gdzies za duzo cofnac podczas debugowania, poprawione